calculator/calculator.go

```diff
@@ -19,3 +19,10 @@ func Divide(a, b int) (int, error) {
 	return a / b, nil
 
 }
+func DividebyZero(a, b int) (c int, ok bool) {
```

Comment:
You need to return an error, not a bool. Do a simple check for the denominator and return an error if `0`, no need to use defer/recover here.